src/linearoperators/DFT.jl

```diff
@@ -32,8 +32,9 @@ julia> A*ones(3)
 struct DFT{N,
 	      C<:RealOrComplex,
 	      D<:RealOrComplex,
+		  E,
 	      T1<:AbstractFFTs.Plan,
-	      T2<:AbstractFFTs.Plan} <: FourierTransform{N,C,D,T1,T2}
+	      T2<:AbstractFFTs.Plan} <: FourierTransform{N,C,D,E,T1,T2}
 	dim_in::NTuple{N,Int}
 	A::T1
 	At::T2
```

Comment:
Just to clarify what I said in the other comment. I think something like this could be more approriate:
```julia
struct DFT{N,
	      C<:RealOrComplex,
	      D<:RealOrComplex,
	      E<:Int,
	      T1<:AbstractFFTs.Plan,
	      T2<:AbstractFFTs.Plan} <: FourierTransform{N,C,D,T1,T2}
	      T2<:AbstractFFTs.Plan} <: FourierTransform{N,C,D,E,T1,T2}
	dim_in::NTuple{N,E}
	A::T1
	At::T2
        dims::E
end
```

Reply:
Yes, much better!